<ui version="4.0" >
 <class>QDialogProgress</class>
 <widget class="QDialog" name="QDialogProgress" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QFormLayout" name="formLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Preparation:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QProgressBar" name="progressPreparation" >
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Printing:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QProgressBar" name="progressPrinting" >
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <layout class="QFormLayout" name="formLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Elapsed time:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QTimeEdit" name="timeElapsed" >
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="buttonSymbols" >
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="keyboardTracking" >
             <bool>false</bool>
            </property>
            <property name="currentSection" >
             <enum>QDateTimeEdit::HourSection</enum>
            </property>
            <property name="displayFormat" >
             <string>hh:mm:ss</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Estimated time:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QTimeEdit" name="timeEstimated" >
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="buttonSymbols" >
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="keyboardTracking" >
             <bool>false</bool>
            </property>
            <property name="currentSection" >
             <enum>QDateTimeEdit::HourSection</enum>
            </property>
            <property name="displayFormat" >
             <string>hh:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Result:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lineResult" >
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Abort</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
